confidant/services/iamrolemanager.py

```diff
@@ -0,0 +1,50 @@
+import gevent
+import logging
+import random
+
+import confidant.clients
+from confidant import settings
+
+ROLES = []
+
+
+def refresh_cache():
+    global ROLES
+    refresh_rate = settings.BACKGROUND_CACHE_IAM_ROLE_REFRESH_RATE
+    if settings.BACKGROUND_CACHE_IAM_ROLE_REFRESH_RATE < 60:
+        refresh_rate = 60
```

Comment:
nit: Should we make the 60 a configurable constant in settings?

Reply:
Eh, nm.  I think okay.  I see there's a  +- 20 secs which makes things a bit more complicated.